app/models/views/applikation/result.rb

```diff
@@ -0,0 +1,28 @@
+module Views
+  module Applikation
```

Comment:
Weird, because everytime I have tried to rename this object to 
```ruby
module Views
  module Application
    class Result < Views::ApplicationResult
```
It bombed out... badly. Never mind tonight, I'll start again in the morning.